### ISSUE TITLE
Bourbon isn't being "automagically" loaded

### ISSUE THREAD

**Issue by Veraxus** [bdkjones/CodeKit2#261]
Bourbon isn't being automatically loaded by CodeKit as in past versions.

**Project Info:**
My project is a Zurb Foundation (Compass version - NOT libsass) WordPress theme with @import "bourbon"; added to app.scss.

CodeKit correctly detects that the project is using Compass, Bourbon, Foundation... but I get an error indicating that Bourbon can't be found.

**Troubleshoot Info:**
If I manually copy the files into my SCSS includes with my other dependencies it works (well, mostly) as expected, but the "automagic" CodeKit handling doesn't appear to be working.

Here's the full error:

```
Compass was unable to compile one or more files in the project: 

    error app.scss (Line 11: File to import not found or unreadable: bourbon.
Load paths:
  /Users/Matt/Dropbox/_Web Projects/artslearning.dev/wp-content/themes/artslearn/_foundation/scss
  /Library/Ruby/Gems/2.0.0/gems/compass-0.12.2/frameworks/blueprint/stylesheets
  /Library/Ruby/Gems/2.0.0/gems/compass-0.12.2/frameworks/compass/stylesheets
  /Users/Matt/Dropbox/_Web Projects/artslearning.dev/wp-content/themes/artslearn/_foundation/bower_components/foundation/scss
  Compass::SpriteImporter)
identical app.css 
```

**Comment by bdkjones** [comment]
Please zip the project as a download and post a link.

I need just the stylesheets this time (the app.scss and any dependencies). Not a full Wordpress dump.

**Comment by Veraxus** [comment]
Here's the WordPress theme project (stripped of PHP and extraneous files)...

https://dl.dropboxusercontent.com/u/8574167/theme%20sass.zip

**Comment by bdkjones** [comment]
The issue is that you're using the `add_import_path` line in your Compass config.rb file. See this issue on Compass --> https://github.com/chriseppstein/compass/issues/1569

**Comment by Veraxus** [comment]
Oh wow, that's a nasty little Compass bug. Thanks.

Fortunately there are workarounds at the moment!

**Comment by bdkjones** [comment]
Fixed in 2.0.1 with a Compass update.
